apps/site/keystatic.config.ts

```diff
@@ -138,6 +138,7 @@ export default config({
               },
             }),
             noTitle: fields.checkbox({ label: 'No title' }),
+            noDemo: fields.checkbox({ label: 'No Demo' }),
```

Comment:
Update the label to 'No demo button on examples' so its a bit clearer what it does for the designers